site/pages/api/create-consequence-operator.test.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-assignment  */
+import { describe, it, expect, afterEach, vi } from "vitest";
+import createConsequenceOperator from "./create-consequence-operator.api";
+import {
+    COOKIES_CONSEQUENCE_OPERATOR_ERRORS,
+    COOKIES_CONSEQUENCE_INFO,
+    CREATE_CONSEQUENCE_OPERATOR_PATH,
+    REVIEW_DISRUPTION_PAGE_PATH,
+    Severity,
+} from "../../constants";
+import { ErrorInfo } from "../../interfaces";
+import { getMockRequestAndResponse } from "../../testData/mockData";
+import { setCookieOnResponseObject } from "../../utils/apiUtils";
+import { ConsequenceOperatorPageInputs } from "../create-consequence-operator.page";
+
+const defaultOperatorData: ConsequenceOperatorPageInputs = {
+    consequenceOperator: "FMAN",
+    description:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    removeFromJourneyPlanners: "no",
+    disruptionDelay: "",
+    disruptionSeverity: Severity.slight,
+    disruptionDirection: "allDirections",
+};
+describe("create-consequence-operator API", () => {
+    const writeHeadMock = vi.fn();
+    vi.mock("../../utils/apiUtils", async () => ({
+        // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+        ...((await vi.importActual("../../utils/apiUtils")) as object),
```

Comment:
remove as object and then remove // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion